python/mxnet/ndarray.py

```diff
@@ -443,12 +458,26 @@ def _slice(self, start, stop):
         return NDArray(handle=handle, writable=self.writable)
 
     def _at(self, idx):
-        """Returns a sliced view of this array.
+        """Returns a view of the array sliced at idx in the first dim.
```

Comment:
explain that this is usually called through x[i]. show this in example.

same for the other apis.